Features/PlayingNow/PlayingNowCollectionViewController.swift

```diff
@@ -8,12 +8,28 @@
 import UIKit
 
 class PlayingNowCollectionViewController: UICollectionViewController {
-    private var movies = [Movie]()
-    private let movieDataService = MovieDataService()
-    private let watchlistService = WatchlistService()
+    private var movieDataService: MovieDataServiceProtocol
+    private var watchlistService: WatchlistServiceProtocol
     
+    private var movies = [Movie]()
     var selectedItemId: Int?
     
+//    Did I get you right when you talked about replacing services with protocols?
+//    I've made changes only on PlayingNoewVC so far.
```

Comment:
Yep, you got it right, that's it 😄
If you want to read more about it, when you set a dependency (the services in this case) using the init it's called `Constructor injection`. There are several ways to achieve it like `Property injection`, `Method injection` or `Service locator`. I can explain them if you want as well.